### ISSUE TITLE
[question] TypeError: render() missing 1 required positional argument: 'mode'

### ISSUE THREAD

**Issue by borninfreedom** [hill-a/stable-baselines#993]
When I run the example atari code like this:
```python
from stable_baselines.common.cmd_util import make_atari_env
from stable_baselines.common.vec_env import VecFrameStack
from stable_baselines import ACER

env=make_atari_env('PongNoFrameskip-v4',num_env=4,seed=0)
env=VecFrameStack(env,n_stack=4)

model=ACER('CnnPolicy',env,verbose=1)
model.learn(total_timesteps=25000)

obs=env.reset()
while True:
    action,_state=model.predict(obs)
    obs,rewards,dones,info=env.step(action)
    env.render()

```
I will encounter the TypeError.
```python
Traceback (most recent call last):
  File "C:\Users\born_\anaconda3\envs\workEnv\lib\site-packages\IPython\core\interactiveshell.py", line 3343, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-2-a12ed3da7d04>", line 1, in <module>
    runfile('C:/Users/born_/PycharmProjects/test_sb/myAtari.py', wdir='C:/Users/born_/PycharmProjects/test_sb')
  File "C:\Program Files\JetBrains\PyCharm 2020.2\plugins\python\helpers\pydev\_pydev_bundle\pydev_umd.py", line 197, in runfile
    pydev_imports.execfile(filename, global_vars, local_vars)  # execute the script
  File "C:\Program Files\JetBrains\PyCharm 2020.2\plugins\python\helpers\pydev\_pydev_imps\_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "C:/Users/born_/PycharmProjects/test_sb/myAtari.py", line 15, in <module>
    env.render()
  File "C:\Users\born_\anaconda3\envs\workEnv\lib\site-packages\stable_baselines\common\vec_env\base_vec_env.py", line 247, in render
    return self.venv.render(*args, **kwargs)
  File "C:\Users\born_\anaconda3\envs\workEnv\lib\site-packages\stable_baselines\common\vec_env\dummy_vec_env.py", line 85, in render
    return super().render(*args, **kwargs)
TypeError: render() missing 1 required positional argument: 'mode'
```

**System Info**
* tensorflow version: 1.15.0 gpu
* gym version: 0.17.2
* os: win10

**Comment by araffin** [comment]
Hello,
please give your SB version next time.
You need to upgrade, this was fixed in #880 .

**Comment by borninfreedom** [comment]
It's 2.10.0
